plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -68,6 +68,14 @@ protected function render( $attributes, $content, $block ) {
 			return '';
 		}
 
+		// We crop the images to square only if the aspect ratio is 1:1.
+		// Otherwise, we show the uncropped and use object-fit to crop them.
+		if ( '1' === $attributes['aspectRatio'] ) {
+			update_option( 'woocommerce_thumbnail_cropping', '1:1' );
+		} else {
+			update_option( 'woocommerce_thumbnail_cropping', 'uncropped' );
+		}
```

Comment:
Is it intentional that we update this option on render? I feel it's a bit weird and might have performance issues. If two Product Gallery Thumbnails blocks on the same page have different `aspectRatio` attributes, that means we would be updating this option one way and the other on every render, no? :thinking: 

If I'm understanding this correctly, I wonder if using the `pre_option_woocommerce_thumbnail_cropping` filter to override the option only when rendering this block might be a good alternative. Or if we want this option to be synced across blocks and all WooCommerce, we could add a global attribute like in https://github.com/woocommerce/woocommerce/pull/54166?